srcpkgs/julia/template

```diff
@@ -1,65 +1,65 @@
 # Template file for 'julia'
 pkgname=julia
-version=1.4.2
+version=1.5.0
 revision=1
-archs="i686* x86_64*"
+archs="i686* x86_64* armv7l* aarch64*"
 build_style=gnu-makefile
 make_build_args="prefix=/usr sysconfdir=/etc datarootdir=/usr/share
- USE_LLVM_SHLIB=1 USE_BINARYBUILDER=0
- USE_SYSTEM_LIBUV=0 USE_SYSTEM_LIBUNWIND=0 USE_SYSTEM_PATCHELF=1 USE_SYSTEM_LIBM=0
- USE_SYSTEM_DSFMT=0 USE_SYSTEM_LLVM=0 USE_SYSTEM_PCRE=1 USE_SYSTEM_BLAS=0
- USE_SYSTEM_GMP=1 USE_SYSTEM_LIBGIT2=0 USE_SYSTEM_MBEDTLS=1 USE_SYSTEM_LIBSSH2=1
- USE_SYSTEM_CURL=1 USE_SYSTEM_MPFR=1 USE_SYSTEM_SUITESPARSE=0 USE_SYSTEM_UTF8PROC=0
- USE_SYSTEM_ZLIB=1 USE_SYSTEM_P7ZIP=1 USE_SYSTEM_LAPACK=0"
+ USE_BINARYBUILDER=0 USE_SYSTEM_LIBM=0 USE_SYSTEM_DSFMT=0 USE_SYSTEM_LIBUV=0
+ USE_SYSTEM_SUITESPARSE=0 USE_SYSTEM_LLVM=0 USE_SYSTEM_BLAS=0 USE_SYSTEM_LAPACK=0
+ USE_SYSTEM_LIBUNWIND=1 USE_SYSTEM_PCRE=1 USE_SYSTEM_GMP=1 USE_SYSTEM_PATCHELF=1
+ USE_SYSTEM_MPFR=1 USE_SYSTEM_MBEDTLS=1 USE_SYSTEM_LIBSSH2=1 USE_SYSTEM_CURL=1
+ USE_SYSTEM_ZLIB=1 USE_SYSTEM_P7ZIP=1 USE_SYSTEM_LIBGIT2=1 USE_SYSTEM_UTF8PROC=1
+ UTF8PROC_INC='${XBPS_CROSS_BASE}/usr/include/libutf8proc'"
 make_install_args="$make_build_args"
 make_check_args="$make_build_args"
 make_check_target=testall
 conf_files="/etc/julia/startup.jl"
-hostmakedepends="perl cmake python gcc-fortran patchelf which tar xz"
-makedepends="pcre2-devel gmp-devel mpfr-devel libcurl-devel
- libssh2-devel mbedtls-devel libatomic-devel zlib-devel p7zip"
-depends="pcre2 gmp mpfr libcurl libssh2 mbedtls libatomic zlib p7zip"
+hostmakedepends="pkg-config perl cmake python gcc-fortran patchelf which tar xz"
+makedepends="p7zip pcre2-devel mpfr-devel libgit2-devel libcurl-devel
+ libssh2-devel mbedtls-devel libatomic-devel zlib-devel libunwind-devel
+ libutf8proc-devel"
+# Julia provides vendored symlinks in /usr/lib/julia pointing to these libraries,
+# but none of the julia executables link to them so these deps can't be detected
+depends="libgit2 libcurl mpfr mbedtls libpcre2 libssh2 p7zip"
```

Comment:
Should I add the specific sonames required to `shlib_requires` so this package gets picked up as requiring a revbump when a dependency soname changes?

Reply:
Yes, that's the right thing to do here.